string action
---
string sensor
float64 reward
string result
